dev/workshop/tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
+  "extends": "./tsconfig.settings",
+  "include": ["./__legacy", "./scripts", "./src", "./test", "./typings", "./vite"],
+  "compilerOptions": {
+    "outDir": "./lib/dts/",
```

Comment:
I think it’s good practice to have, even though we‘re not building the TS files in this project (dev/workshop) – because if I’d accidentally run `tsc` the definition files will be written to the all kinds of unwanted places.

Reply:
Good point, agreed.